src/features/device-state/index.ts

```diff
@@ -24,7 +24,10 @@ export const setup = (app: Application) => {
 };
 
 export interface Events {
-	'get-state': (uuid: string) => void;
+	'get-state': (
+		uuid: string,
+		req: Pick<Request, 'apiKey' | 'prefetchApiKey' | 'user'>,
```

Comment:
Is there any reason for exporting all of these rather than just `apiKey`?

Reply:
Picking just the apiKey instead of all the possible ways that the request might be authenticated, felt a bit like hardcoding the pub-sub implementation for the heartbeat needs. Will change to just pick the `apiKey` and we can extend later if needed :+1: